src/callbacks/gradient_clipping.cpp

```diff
@@ -158,7 +158,7 @@ struct NormComputer
         }
 #else
         local_norm = El::Nrm2(gradmatrix);
-#endif
+#endif // LBANN_HAS_HALF
```

Comment:
```suggestion
#endif // LBANN_HAS_GPU_FP16
```
Also, the `#ifdef` line above should be changed accordingly.